configure.ac

```diff
@@ -71,38 +71,17 @@ AC_CHECK_DECLS([_NSGetEnviron],,,[[#include <crt_externs.h>]])
 GEANY_CHECK_REVISION([dnl force debug mode for a VCS working copy
 					  CFLAGS="-g -DGEANY_DEBUG $CFLAGS"])
 
-# GTK version check
-AC_ARG_ENABLE([gtk2],
-		[AS_HELP_STRING([--enable-gtk2],
-						[compile against deprecated GTK2 [default=no]])],
-		[enable_gtk2=$enableval],
-		[enable_gtk2=no])
-
-gtk2_package=gtk+-2.0
-gtk2_min_version=2.24
-gtk3_package=gtk+-3.0
-gtk3_min_version=3.0
-
-AS_IF([test "x$enable_gtk2" = "xyes"],
-	[gtk_package=$gtk2_package
-	 gtk_min_version=$gtk2_min_version],
-	[gtk_package=$gtk3_package
-	 gtk_min_version=$gtk3_min_version])
-
-AM_CONDITIONAL([GTK3], [test "x$gtk_package" = "x$gtk3_package"])
-
```

Comment:
I would rather keep some skeleton of multiple GTK version support to make it easier to support e.g. GTK4 in the future, as mentioned in https://github.com/geany/geany/pull/2617#issuecomment-706535227

Reply:
No need when the build moves to meson </stir>